<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>下位机</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>40</y>
     <width>361</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="3">
     <widget class="QLineEdit" name="Rsend_2">
      <property name="text">
       <string>7.5</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="Rsend">
      <property name="text">
       <string>Rsend</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="longitude">
      <property name="text">
       <string>经度</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="latitude_2"/>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="latitude">
      <property name="text">
       <string>纬度</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="longitude_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="Lsend">
      <property name="text">
       <string>Lsend</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="Lsend_2">
      <property name="text">
       <string>7.5</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="compass">
      <property name="text">
       <string>罗盘</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="compass_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>441</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="3" column="0">
     <widget class="QLabel" name="COM_4">
      <property name="text">
       <string>罗盘 COM</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="COM_3">
      <property name="text">
       <string>GPS COM</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox_2"/>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="comboBox_4"/>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox_1"/>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboBox_3"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="COM_1">
      <property name="text">
       <string>左电机 COM</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="COM_2">
      <property name="text">
       <string>右电机 COM</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QCheckBox" name="checkBox_1">
      <property name="text">
       <string>打开连接</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QCheckBox" name="checkBox_2">
      <property name="text">
       <string>打开连接</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QCheckBox" name="checkBox_3">
      <property name="text">
       <string>打开连接</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QCheckBox" name="checkBox_4">
      <property name="text">
       <string>打开连接</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="checkBox_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>上下位机开始通信</string>
   </property>
  </widget>
  <widget class="QPushButton" name="load_com">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>加载com口</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
